Makefile.toml

```
@@ -34,4 +34,4 @@ env = { "RUSTDOCFLAGS" = "-Dwarnings" }
 [tasks.release]
 description = "Task to release the package to crates.io"
 command = "cargo"
-args = ["release", "publish", "--no-confirm", "--allow-branch \"*\"", "--all-features", "--execute"]
+args = ["release", "publish", "--no-verify", "--no-confirm", "--allow-branch \"*\"", "--all-features", "--execute"]
```

```suggestion
args = ["publish", "--no-verify"]
```

cargo release is a separate binary that wraps cargo publish, we use it in veil since it's made up of multiple crates, but in the case of jwks_client just doing `cargo publish` should be good enough